editor/editor_node.cpp

```diff
@@ -3411,10 +3411,11 @@ void EditorNode::_update_recent_scenes() {
 void EditorNode::_quick_opened() {
 	Vector<String> files = quick_open->get_selected_files();
 
+	bool open_scene_dialog = quick_open->get_base_type() == "PackedScene";
 	for (int i = 0; i < files.size(); i++) {
 		String res_path = files[i];
 
-		if (quick_open->get_base_type() == "PackedScene") {
+		if (open_scene_dialog || files[i].get_extension() == "tscn") {
```

Comment:
There are more extensions that can be used by PackedScene: `scn`, `res`, `tres`.

You should indeed query the ResourceLoader for the extensions supported by PackedScene:
```
Vector<String> scene_extensions;
ResourceLoader::get_recognized_extensions_for_type("PackedScene", scene_extensions);
```

Reply:
I did that initially, but then I saw ResourceLoader loops through a list of 20 something ResourceFormats. But I will change it and look into it later.